GCRCatalogs/catalog_configs/cosmoDC2_v1.1.4_redmapper_v0.5.7.yaml

```
@@ -21,4 +21,4 @@ description: |
   Cluster and member catalogs computed from the redMaPPer cluster finder
   (v0.5.7) run on cosmoDC2 v1.1.4.
 
-include_in_default_catalog_list: true
+include_in_default_catalog_list: false
```

Should this catalog also be deprecated? If so, add the "deprecated" flag:

```suggestion
include_in_default_catalog_list: false
deprecated: Use cosmoDC2_v1.1.4_redmapper_v0.7.5 instead
```